smartapps/juano2310/jawbone-up-connect.src/jawbone-up-connect.groovy

```diff
@@ -287,9 +279,10 @@ def validateCurrentToken() {
 }
 
 def initialize() {
-	def hookUrl = "${serverUrl}/api/token/${state.accessToken}/smartapps/installations/${app.id}/hookCallback"
+    log.debug "Callback URL - Webhook"
+	def localServerUrl = getApiServerUrl()
+	def hookUrl = "${localServerUrl}/api/token/${state.accessToken}/smartapps/installations/${app.id}/hookCallback"
```

Comment:
why can't we use `buildActionUrl("hookCallback")` here, instead of building it manually?